stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -38,6 +38,9 @@ public final class ServerCalls {
   @VisibleForTesting
   static final String MISSING_REQUEST = "Half-closed without a request";
 
+  static final Metadata.Key<String> GRPC_PREVIOUS_RPC_ATTEMPTS =
```

Comment:
It will be nice to have a single definition shared with `RetriableStream` . The current one `RetriableStream.GRPC_PREVIOUS_RPC_ATTEMPTS` can be moved up to the common base class `Stream` then we can eliminate this?